docs/en/concepts-and-designs/probe-introduction.md

```diff
@@ -20,10 +20,10 @@ You don't need to use **Language based native agent** and **Service Mesh probe**
 metric data. As a result of that, your system suffers twice payloads, and the analytic numbers are doubled.
 
 There are several recommend ways in using these probes:
-1. Use **Language based native agent** only.
-1. Use **3rd-party instrument library** only, like Zipkin instrument ecosystem.
-1. Use **Service Mesh probe** only.
-1. Use **Service Mesh probe** with **Language based native agent** or **3rd-party instrument library** in tracing status. (Advanced usage)
+- Use **Language based native agent** only.
+- Use **3rd-party instrument library** only, like Zipkin instrument ecosystem.
+- Use **Service Mesh probe** only.
+- Use **Service Mesh probe** with **Language based native agent** or **3rd-party instrument library** in tracing status. (Advanced usage)
```

Comment:
These are typical ways, why change from `1,2,3,4` to this?

Reply:
Sorry I forgot that GitHub can render the numbers correctly even they were all 1's in the markdown, have changed back.